### PR TITLE
Cleanup alloc

### PR DESCRIPTION
**Pull request by cipriane**
Cleaned up some code & made some braces conform to the style guidelines.

### CODE REVIEW

File: subs/utilities/h/Alloc_m/Alloc_m.h [hunk 1]
```diff
@@ -3,28 +3,29 @@
 
 #include <stdlib.h>
 #include <assert.h>
-//#include "../utlHeavy.h"
 
-class makeList {
+class makeList
+{
```
Comment (chadpaine) [inline subs/utilities/h/Alloc_m/Alloc_m.h]: I love and hate you at the same time.

File: subs/utilities/h/Alloc_m/Alloc_m.h [hunk 2]
```diff
@@ -3,28 +3,29 @@
...
-		
+
 	private:
-		
+
 		template<typename T>
-		struct makeNode {
-			
+		struct makeNode
+		{
```
Comment (chadpaine) [inline subs/utilities/h/Alloc_m/Alloc_m.h]: asdfjlaosdafsdf

File: subs/utilities/h/Alloc_m/Alloc_m.h [hunk 3]
```diff
@@ -3,28 +3,29 @@
...
 
-class makeList {
+class makeList
+{
 	public:
-		
+
 	private:
-		
+
```
Comment (chadpaine) [inline subs/utilities/h/Alloc_m/Alloc_m.h]: why

File: subs/utilities/h/Alloc_m/Alloc_m.h (hunk 1 above)
Comment (cipriane) [reply subs/utilities/h/Alloc_m/Alloc_m.h]: Just following the agreed-upon guidelines.

File: subs/utilities/h/Alloc_m/Alloc_m.h (hunk 3 above)
Comment (cipriane) [reply subs/utilities/h/Alloc_m/Alloc_m.h]: Superfluous whitespace.

File: subs/utilities/h/Alloc_m/Alloc_m.h (hunk 2 above)
Comment (cipriane) [reply subs/utilities/h/Alloc_m/Alloc_m.h]: dfky

File: subs/utilities/h/Alloc_m/Alloc_m.h (hunk 1 above)
Comment (chadpaine) [reply subs/utilities/h/Alloc_m/Alloc_m.h]: MUHAHA

Comment (chadpaine) [reply subs/utilities/h/Alloc_m/Alloc_m.h]: This will be reviewed by management.

File: subs/utilities/h/Alloc_m/Alloc_m.h (hunk 3 above)
Comment (chadpaine) [reply subs/utilities/h/Alloc_m/Alloc_m.h]: Why not

**Comment by cipriane** [comment]
Please accept ASAP

**Pull request merged**